include/solvers/postprocessors.h

```diff
@@ -308,21 +438,43 @@ class DynamicViscosityPostprocessor : public DataPostprocessorScalar<dim>
   }
 
 private:
+  /// Model describing the fluid's rheology.
```

Comment:
Should we not use /*
*
/*

Or you can use the ///?

Reply:
`///` works also! It takes less space for single line descriptions.
![image](https://github.com/lethe-cfd/lethe/assets/107414376/0b0ad55a-bbed-46f3-8bcf-3e210e6b7747)